test/utils.js

```diff
@@ -2,7 +2,8 @@ const unified = require('unified');
 const stringify = require('remark-stringify');
 const markdown = require('remark-parse');
 const html = require('remark-html');
-const {redact, restore, plugins} = require('remark-redactable');
+const {redact, restore, findRestorations} = require('remark-redactable');
+const rawtext = require('../src/rawtext');
```

Comment:
I assume this is necessary because of https://github.com/code-dot-org/remark-redactable/pull/6#discussion_r315869212?

Reply:
I believe so. I'll address that comment in that PR.

Reply:
Hmm maybe not. I added a compiler plugin and still needed `rawtext`